com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -546,40 +548,72 @@ public int additionalLightsShadowmapResolution
             get { return (int)m_AdditionalLightsShadowmapResolution; }
         }
 
+        /// <summary>
+        /// Controls the distance the shadows are cast.
+        /// </summary>
         public float shadowDistance
         {
             get { return m_ShadowDistance; }
             set { m_ShadowDistance = Mathf.Max(0.0f, value); }
         }
 
+        /// <summary>
+        /// Controls the amount of cascades. 0, 2, 3 and 4.
```

Comment:
```suggestion
        /// Controls the amount of cascades. 1, 2, 3 and 4.
```

Reply:
1 == no cascades, it's confusing actually if this should be zero or 1 as the enum is none. Anyway. No strong opinion here.

Reply:
I will change this in next update to 1, so we are in parity with HDRP. 
The enum will also change to a slider like in HDRP and have 1,2,3,4.